Makefile

```diff
@@ -34,7 +34,7 @@ tests: require-env-ghtoken set-build-info
 	docker compose down --remove-orphans && \
 	docker compose build app && \
 	docker compose run app \
-		pytest -vv -s --durations=20 conbench/tests/
+		pytest -vv --durations=20 conbench/tests/
```

Comment:
This is just a preference thing, but I noticed that when we printed the stdout while running the tests, it didn't print it next to the traceback when the tests failed. This is cleaner to me and makes more sense